ow_dynamic_terrain/src/DynamicTerrainBase.cpp

```diff
@@ -37,6 +37,9 @@ void DynamicTerrainBase::Initialize(const std::string& topic_extension)
   subscribe<modify_terrain_patch>("modify_terrain_patch", on_modify_terrain_patch);
   subscribe<modify_terrain_patch>("modify_terrain_patch/" + topic_extension, on_modify_terrain_patch);
 
+  m_differential_pub = m_node_handle->advertise<modified_terrain_diff>(
+      "/" + m_package_name + "/modification_differential/" + topic_extension, 1);
```

Comment:
What is the purpose of adding it at the base class here when you only intend to use one of the two topics (the "visual")? Is it to have the option to alternate between the two?